### ISSUE TITLE
&sketch=1  is ignored

### ISSUE THREAD

**Issue by abdulhakim-alshanqiti** [Watt3r/d2-live#4]
https://play.d2lang.com/?script=qrBSCMlILFEvVkjPL1EoyVdISlUoyUhVSEotBnHzcxQ81ctSFVLLUosUilNT87gAAQAA__8%3D&sketch=1&theme=200&

![very nice](https://d2.atlas.lucas.tools/?script=qrBSCMlILFEvVkjPL1EoyVdISlUoyUhVSEotBnHzcxQ81ctSFVLLUosUilNT87gAAQAA__8%3D&sketch=1&theme=200&)

this should be in sketch 👌this is the final detail i think for the tool to have 1 to 1 mapping with the playground

**Comment by oberwager** [comment]
Yeah I can look into this in a week or so, feel free to open a PR if you want

**Comment by abdulhakim-alshanqiti** [comment]
✨👌 no problem i would have made a pr if i knew go

don't feel like you need to rush to fix this

**Comment by oberwager** [comment]
Fixed in 2341157
